server/src/internalClusterTest/java/org/opensearch/cluster/coordination/UnsafeBootstrapAndDetachCommandIT.java

```diff
@@ -34,31 +33,40 @@
 import org.opensearch.gateway.GatewayMetaState;
 import org.opensearch.gateway.PersistedClusterStateService;
 import org.opensearch.indices.IndicesService;
-import org.opensearch.test.OpenSearchIntegTestCase;
 import org.opensearch.test.InternalTestCluster;
+import org.opensearch.test.OpenSearchIntegTestCase;
 
 import java.io.IOException;
 import java.util.ArrayList;
 import java.util.List;
 import java.util.Locale;
+import java.util.Objects;
 
+import static org.hamcrest.Matchers.containsString;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.notNullValue;
 import static org.opensearch.action.support.WriteRequest.RefreshPolicy.IMMEDIATE;
 import static org.opensearch.gateway.DanglingIndicesState.AUTO_IMPORT_DANGLING_INDICES_SETTING;
 import static org.opensearch.index.query.QueryBuilders.matchAllQuery;
 import static org.opensearch.indices.recovery.RecoverySettings.INDICES_RECOVERY_MAX_BYTES_PER_SEC_SETTING;
 import static org.opensearch.test.NodeRoles.nonMasterNode;
+import static org.opensearch.test.hamcrest.OpenSearchAssertions.assertAcked;
 import static org.opensearch.test.hamcrest.OpenSearchAssertions.assertHitCount;
-import static org.hamcrest.Matchers.containsString;
-import static org.hamcrest.Matchers.equalTo;
-import static org.hamcrest.Matchers.notNullValue;
 
 @OpenSearchIntegTestCase.ClusterScope(scope = OpenSearchIntegTestCase.Scope.TEST, numDataNodes = 0, autoManageMasterNodes = false)
 public class UnsafeBootstrapAndDetachCommandIT extends OpenSearchIntegTestCase {
 
-    private MockTerminal executeCommand(OpenSearchNodeCommand command, Environment environment, int nodeOrdinal, boolean abort)
-            throws Exception {
+    private MockTerminal executeCommand(OpenSearchNodeCommand command, Environment environment, int nodeOrdinal,
+                                        boolean abort, Boolean applyClusterReadOnlyBlock)
```

Comment:
1. For java doc I am not able to find any documentation in UnsafeBootstrapMasterCommand.java file. But if user gives --help argument in command then it will show about this newly added argument and it's default value.
```
╰$ ./opensearch-node unsafe-bootstrap --help 
warning: no-jdk distributions that do not bundle a JDK are deprecated and will be removed in a future release
Forces the successful election of the current node after the permanent loss of the half or more master-eligible nodes

Option                           Description                      
------                           -----------                      
-E <KeyValuePair>                Configure a setting              
--apply-cluster-read-only-block  Optional cluster.blocks.read_only
  [Boolean]                        setting, false if not specified
-h, --help                       Show help                        
--ordinal <Integer>              Optional node ordinal, 0 if not  
                                   specified                      
-s, --silent                     Show minimal output              
-v, --verbose                    Show verbose output 
```
2. Yes if cx needs to restore from snapshot then he/she needs to remove this read block.